presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -268,6 +268,21 @@ else if (varcharType.getBoundedLength() <= 16777215) {
         return super.toWriteMapping(session, type);
     }
 
+    private boolean isGTIDMode(Connection connection)
```

Comment:
Move this method to after `jsonColumnMapping` method. 
```suggestion
    private boolean isGtidMode(Connection connection)
```